mod/member_selfdelete/manifest.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8"?>
<plugin_manifest xmlns="http://www.elgg.org/plugin_manifest/1.8">
    <name>Member Self Delete</name>
    <id>member_selfdelete</id>
    <author>Matt Beckett</author>
    <version>2.0</version>
    <description>Allows members to delete their account, and submit an optional reason for deleting their account</description>
    <website>http://mattbeckett.me/</website>
    <copyright>See COPYRIGHT.txt</copyright>
    <license>GNU General Public License version 2</license>
    <requires>
        <type>elgg_release</type>
        <version>1.9</version>
    </requires>
</plugin_manifest>
```

Comment:
Seems to have odd line endings here, though if it doesn't prevent the mod from activating it shouldn't really matter.